Cargo.toml

```diff
@@ -19,11 +19,16 @@ members = [ "io-uring-test", "io-uring-bench" ]
 
 [dependencies]
 bitflags = { version = "2.4.0", default-features = false }
-rustix = { version = "1.0.2", default-features = false, features = ["io_uring", "mm", "thread"] }
+linux-raw-sys = { version = "0.10.0", default-features = false }
```

Comment:
Lets try and avoid adding any dependencies. This should really just be rustix + any supporting helpers.